pkg/operator/ceph/object/spec.go

```diff
@@ -381,8 +383,36 @@ func (c *clusterConfig) reconcileExternalEndpoint(cephObjectStore *cephv1.CephOb
 	return nil
 }
 
+func (c *clusterConfig) generateRoute(cephObjectStore *cephv1.CephObjectStore) (*routev1.Route, error) {
+	route := &routev1.Route{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      instanceName(cephObjectStore.Name),
+			Namespace: cephObjectStore.Namespace,
+		},
+		Spec: routev1.RouteSpec{
+			To: routev1.RouteTargetReference{
+				Kind: "Service",
+				Name: "rook-ceph-rgw-my-store",
```

Comment:
Is this name going to be the same for all CephObjectStores?

Reply:
`Name: "rook-ceph-rgw-my-store", `# The name of the RGW service is in the form `rook-ceph-rgw-<objectstore-name>`. see [this](https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/object-openshift.yaml#L108)